.github/workflows/update-matrix.yml

```diff
@@ -0,0 +1,27 @@
+name: Update Compatibility Matrix
+
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  update_issue_number:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Find and Replace
+        run: |
+          sed -i "s/implemented: false, issue: ${GITHUB_EVENT_ISSUE_NUMBER}/implemented: true/g" docs/04-reference/compatibility/compatibility.json
```

Comment:
Maybe we can find some way to still store the issue number in the table (separate from whether or not we're showing a checkmark) so that the compatibility matrix will also be updated if an issue is re-opened